packages/components/src/Layout/OneColumn/OneColumn.component.js

```diff
@@ -20,12 +20,12 @@ function OneColumn({ drawers, children, tabs, ...props }) {
 		flexDirection: 'column',
 	};
 	return (
-		<div className={container} {...props}>
+		<main role="main" className={container} {...props}>
```

Comment:
I didn't knew about this tag :+1: 
but what about the fact that there can't be several < main > in a document as we can have several layouts with one column...

Reply:
We don't care, there is no style related